tools/get_config.py

```diff
@@ -36,19 +37,15 @@
 if __name__ == '__main__':
     # Parse Options
     parser = get_default_options_parser(add_clean=False, add_options=False)
-    parser.add_option("--source", dest="source_dir",
-                      default=None, help="The source (input) directory", action="append")
-    parser.add_option("--prefix", dest="prefix", action="append",
-                      default=None, help="Restrict listing to parameters that have this prefix")
-    parser.add_option("-v", "--verbose", action="store_true", dest="verbose",
+    parser.add_argument("--source", dest="source_dir", type=argparse_filestring_type,
```

Comment:
If a `--source` argument isn't specified, then I just get an ERROR:

```
$ python tools\get_config.py -m K64F -t GCC_ARM
[ERROR] list index out of range
```

@bogdanm Does it ever make sense to run this script without the `--source` option specified? If not then we should probably make this a required argument.